### PR TITLE
Add Google Assistant integration

### PR DESCRIPTION
**Pull request by balloob**
It's time 👍

### CODE REVIEW

File: panels/config/cloud/ha-config-cloud-account.html [hunk 1]
```diff
@@ -95,16 +96,39 @@
             <div class='card-actions'>
               <a href='https://alexa.amazon.com/spa/index.html#skills/dp/B0772J1QKB/?ref=skill_dsk_skb_sr_2' target='_blank'>
                 <paper-button>
-                  Home Assistant Alexa skill
+                  Go to Alexa skill
                 </paper-button>
               </a>
-              <a href='https://home-assistant.io/components/cloud.alexa/' target='_blank'>
+              <a href='https://home-assistant.io/cloud/alexa/' target='_blank'>
```
Comment (c727) [inline panels/config/cloud/ha-config-cloud-account.html]: www.

File: panels/config/cloud/ha-config-cloud-account.html [hunk 2]
```diff
@@ -95,16 +96,39 @@
...
+              With the Google Assistant integration for Home Assistant Cloud you'll be able to control all your Home Assistant devices via any Google Assistant-enabled device.
+              <p><em>This integration requires a Google Assistant-enabled device like the Google Home or an Android phone.</em></p>
+            </div>
+            <div class='card-actions'>
+              <a href='https://assistant.google.com/services/a/uid/00000091fd5fb875' target='_blank'>
+                <paper-button>
+                  Go to Google Assistant skill
+                </paper-button>
+              </a>
+              <a href='https://home-assistant.io/cloud/google_assistant/' target='_blank'>
```
Comment (c727) [inline panels/config/cloud/ha-config-cloud-account.html]: www.

**Comment by SamuiSmartHome** [comment]
This is fantastic. Waiting for updated.

**Review by c727 — COMMENTED** [review]
Here we also have mixed types of card actions. Also we should have only 2 text action buttons in a card

**Comment by balloob** [comment]
Fixed

image

**Pull request merged**